zadanie3/zadanie3.hpp

```diff
@@ -4,11 +4,20 @@
 struct Element {
     int skrzynka;
     Element* next;
+
+    ~Element(){
+        if(next) delete next;
```

Comment:
if niekoniecznie potrzebny, zgodnie ze specyfikacja jak sie nie uzywa swoich managerow pamieci to delete na `nullptr` powinno nie robic nic, ale dodalem na wszelki wypadek